rector_examples/test/src/Functional/BrowserTestBaseGetMock.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Drupal\Tests\rector_examples\Functional;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Tests\BrowserTestBase;
+
+class BrowserTestBaseGetMock extends BrowserTestBase {
+
+  /**
+   *
+   */
```

Comment:
@shaal , can you add comments for these examples?

Reply:
👍